civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -1680,9 +1671,50 @@ function civihr_employee_portal_menu() {
     'type' => MENU_NORMAL_ITEM,
   ];
 
+  // Create Emergency Contact webform ajax popup
+  $items['create-emergency-contact/js/view'] =
+    _civihr_employee_portal_get_webform_modal_menu_item('Create Emergency Contact');
+
+  // Create Dependant webform ajax popup
+  $items['create-dependant/js/view'] =
+    _civihr_employee_portal_get_webform_modal_menu_item('Create Dependant');
+
+  // My Details - My Home Address webform ajax popup
+  $items['edit-my-home-address/js/view'] =
+    _civihr_employee_portal_get_webform_modal_menu_item('My Home Address');
+
+  // My Details - My Payroll webform ajax popup
+  $items['edit-my-payroll/js/view'] =
+    _civihr_employee_portal_get_webform_modal_menu_item('My Payroll');
+
+  // My Details - Personal Details webform ajax popup
+  $items['edit-my-personal-details/js/view'] =
+    _civihr_employee_portal_get_webform_modal_menu_item('My Personal Details');
+
+  // My Details - Contact Info webform ajax popup
+  $items['edit-my-contact-details/js/view'] =
+    _civihr_employee_portal_get_webform_modal_menu_item('My Contact Details');
+
   return $items;
 }
 
+/**
+ * To create menu items for webform modals
+ *
+ * @param array $items menu items reference
+ *
+ * @return array the structure of the menu item for webform popup
+ */
+function _civihr_employee_portal_get_webform_modal_menu_item($name) {
+  // Base structure for a menu item which renders a webform asynchronously
```

Comment:
I think this comment is not necessary, you are already mentioning the same in `@return array the structure of the menu item for webform popup`